taxi_service/settings.py

```diff
@@ -124,6 +124,10 @@
 
 STATIC_URL = "static/"
 
+STATICFILES_DIRS = [
+    BASE_DIR / "static"
+]
```

Comment:
The STATICFILES_DIRS setting is correctly configured to include the 'static' directory within the BASE_DIR. Ensure that the directory exists and contains your static files.